NextStepWeek1Calculator/src/com/nextstep/calculator/Calculator.java

```diff
@@ -0,0 +1,46 @@
+package com.nextstep.calculator;
+
+public class Calculator {
+
+	int Add(String input) {
+		int result = 0;
+		if(input.isEmpty() || input.isBlank())
+		{
+			return result;
```

Comment:
```suggestion
			return 0;
```
[제안] 특정 상황에서 result라는 값을 반환해 주는 것보다 0을 반환하므로써 result라는 변수가 변경될 수 있는 상황을 방지해 주는 것은 어떨까요? 
더 나아가서 0을 상수화 시키는 것도 괜찮을 것 같아요

result와 if 문사이에 로직이 들어간다면, result의 값은 변경의 여지가 있어보여요